main.go

```diff
@@ -96,14 +117,32 @@ func main() {
 	flag.StringVar(&provenancePath, "provenance", "", "path to a provenance file")
 	flag.StringVar(&binaryPath, "binary", "", "path to a binary to verify")
 	flag.StringVar(&source, "source", "", "expected source repository that should have produced the binary, e.g. github.com/some/repo")
-	flag.StringVar(&branch, "branch", "main", "expected branch the binary was compiled from. Default: main")
+	flag.StringVar(&branch, "branch", "main", "[optional] expected branch the binary was compiled from")
```

Comment:
remove the default param for main here

Reply:
why? Is this not supposed to be the default branch we should verify by default? In fact if we want that my code is wrong because it uses a pointer :-)

Reply:
I guess you mean this should not be an option paramater?

Reply:
Yeah, it's confusing it's optional but has a default. maybe just specify that this will be default main and you don't need to check if it's nil